core/embed/bootloader/main.c

```diff
@@ -404,6 +408,7 @@ int bootloader_main(void) {
   random_delays_init();
 
 #ifdef STM32U5
+
```

Comment:
Unnecessary nested #ifdef STM32U5

Reply:
yes. i will be moving the sec area configuration to boardloader anyway in subsequent PR, so this code will be somewhat simplified. I would keep this as is until then as this was not modified here (apart for adding a blank line which is removed in this fixup: https://github.com/trezor/trezor-firmware/commit/e419986e6222fcf7b01fd59b68cd6d8f7e82465b )